modules/settings/tfe_base_config.tf

```
@@ -36,8 +36,8 @@ locals {
       value = random_id.cookie_hash.hex
     }
 
-    consolidated_services = {
-      value = var.consolidated_services != null ? var.consolidated_services ? "1" : "0" : null
+    consolidated_services_enabled = {
+      value = var.consolidated_services_enabled != null ? var.consolidated_services_enabled ? "1" : "0" : null
```

```suggestion
      value = var.consolidated_services_enabled == null ? null : var.consolidated_services_enabled ? "1" : "0"
```

Positive conditionals are easier to reason with 👍🏼

---

I agree with you, but we discovered early on when creating this module that some of the values wouldn't compile properly, and we switched them all to use negative conditionals as a standard.